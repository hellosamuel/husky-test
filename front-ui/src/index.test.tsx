import { render } from '@testing-library/react'
import App from './App'

test('Render App correctly', () => {
  const { asFragment } = render(<App />)

  expect(asFragment()).toMatchSnapshot()
})
