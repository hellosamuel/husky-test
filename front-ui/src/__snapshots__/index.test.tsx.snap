// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <img
        alt="logo"
        class="App-logo"
      />
      <p>
        Edit 
        <code>
          src/App.tsx
        </code>
         and save to reload.
      </p>
      <a
        class="App-link"
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        Learn React
      </a>
    </header>
  </div>
</DocumentFragment>
`;
